getting-started/quick-start.md

```diff
@@ -0,0 +1,113 @@
+cd amber-stable/
+shards install
+make install
+```
+
+#### Full Install Instructions
+See full installation instructions [here](https://amberframework.org/guides/getting-started/Installation/README.md#installation) for RedHat & CentOS, ArchLinux & Derivatives, and more complete instructions.
+
+### Install a database
+Amber works with `postgresql` (default), `mysql`, or `sqlite`.
+
+If you don’t already have one of these installed, please follow the guides provided by each database maintainer:
+[Postgresql Installation Guides](https://wiki.postgresql.org/wiki/Detailed_installation_guides)
+On OS X postgres can also be installed with homebrew or the Postgres.app
+[MySQL Installation Guides](https://dev.mysql.com/doc/refman/8.0/en/installing.html)
+On OS X mysql can also be installed with homebrew
+[Tutorial on installing SQLite](https://www.tutorialspoint.com/sqlite/sqlite_installation.htm)
+On OS X sqlite3 can also be installed with homebrew
+
+## Generate a new Amber application
+With all dependencies successfully installed, we can generate a new application with `amber new`
+
+After the code for the new application is generated, we will `cd` into the new directory and execute a `shards install`.
+
+The shards install command may take a little while - it has to download all shard dependencies.
+```
+# The default setup will use a postgresql database, use `-d mysql` or `-d sqlite` for mysql and sqlite, respectively
+$ amber new pet-tracker
+[output of all the generated files]
+$ cd pet-tracker
+$ shards install
+[output of shard dependencies being installed]
+```
+
+## Generate a resource
+With the skeleton application generated, we can generate our first RESTful resource.
+
+ The `amber generate scaffold` command will help us do this.
+
+```
+$ amber generate scaffold Pet name breed age:integer weight:integer
+[output of scaffold code generated]
+```
+
+## Create and migrate the database
+Generating the application and the scaffolded resource provides the configuration and migration files needed to setup our database.
+
+`amber db` will help us do this.
+```
+$ amber db create migrate
+[output of database operations]
+```
+This will create a new database and run the migration to create a `pets` table with the specified columns.
+
+## Build the application and run the server
+We can use `amber watch` to both build the binary application and start the server. Additionally, `amber watch` will detect code changes then recompile and restart the application automatically.
+```
+amber watch
+```
+
+## Use your brand new web application!
+Open any browser and goto http://localhost:3000
+You should see a home page load and “Pets” in the nav bar.
+If you click on the “Pets” link, you should be able to perform all 7 RESTful actions for the “pets” resource.
+
+## List of Commands
+```
+amber new pet-tracker
+cd pet-tracker
+shards install
+amber generate scaffold Pet name breed age:integer weight:integer
+amber db create migrate
+amber watch
+```
```

Comment:
should we add an optional section regarding assets: `npm install && npm watch`?  I know we want to add this to `amber watch` but I think if we don't mention it, people will get the impression this is handled.

Reply:
On the next release `amber w` will automatically compile assets. See https://github.com/amberframework/amber/pull/451

Reply:
@drujensen - We could possible link to that, but I don't want to introduce an npm dependency to this page.

The idea of the quick-start guide is to be the quickest way to get up and running - far from all-inclusive.